components/twitter/twitter.app.js

```diff
@@ -818,19 +885,212 @@ module.exports = {
         console.log(err) // TODO
       }
     },
-    async search(q, since_id, tweet_mode, count, result_type, lang, locale, geocode) {   
-      const query = querystring.stringify({
-        q,
-        since_id,
-        tweet_mode,
-        count,
+    async getFollowers(screen_name) {   
+      return (await this._makeRequest({
+        url: `https://api.twitter.com/1.1/followers/ids.json?`,
+        params: {
+          screen_name,
+          stringify_ids: true,
+        }
+      })).data.ids
+    },
+    async getLikedTweets(opts = {}) {
+      const {
+        screen_name,
+        count = 200,
+        tweet_mode = 'extended',
+      } = opts
+      return (await this._makeRequest({
+        url: `https://api.twitter.com/1.1/favorites/list.json`,
+        params: {
+          screen_name,
+          count,
+          tweet_mode,
+        }
+      })).data
+    },
+    async lookupUsers(userIdArray) {   
+      return (await this._makeRequest({
+        url: `https://api.twitter.com/1.1/users/lookup.json`,
+        params: {
+          user_id: userIdArray.join(),
```

Comment:
Would I had no idea and would not have predicted the default separator to be a comma.